src/plugins/console/index.js

```diff
@@ -88,10 +87,7 @@ module.exports = function (kibana) {
       const proxyRouteConfig = {
         validate: {
           query: Joi.object().keys({
-            uri: Joi.string().uri({
-              allowRelative: false,
-              shema: ['http:', 'https:'],
-            }),
+            uri: Joi.string()
```

Comment:
Mostly to ensure that an empty string is not passed in (see https://github.com/hapijs/joi/blob/v8.0.5/API.md#string)